src/v/kafka/server/quota_manager.h

```diff
@@ -85,26 +91,46 @@ class quota_manager {
       uint64_t bytes,
       clock::time_point now = clock::now());
 
-private:
-    // erase inactive tracked quotas. windows are considered inactive if they
-    // have not received any updates in ten window's worth of time.
-    void gc(clock::duration full_window);
+    // record new desired number of partition mutations
+    ss::future<std::chrono::milliseconds> record_partition_mutations(
```

Comment:
since this hops cores to update usage, it probably only makes sense for low-traffic ops (like in this PR). where can we add a comment of some sort that it isn't appropriate for high-freq ops like produce or other things?

Reply:
Yeah sure, good idea